draws/db.py

```
@@ -0,0 +1,212 @@
+        return []
+
+def get_next_pending_draw(category_id: Optional[str] = None) -> Draw | None:
+    """
+    Retrieves the next draw with status 'pending_open', ordered by scheduled_open_time.
+    Optionally filters by category_id.
+    """
+    try:
+        collection = get_draws_collection()
+        query: Dict[str, Any] = {"status": "pending_open"}
+        if category_id:
+            if not ObjectId.is_valid(category_id): # Basic check, though category_id is str
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)